cmd/earthly/root_cmds.go

```diff
@@ -290,8 +290,8 @@ Set up a whole custom git repository for a server called example.com, using a si
 		{
 			Name:        "registry",
 			Aliases:     []string{"registries"},
-			Description: "Manage registry access *experimental*",
-			Usage:       "Manage registry access *experimental*",
+			Description: "Manage registry access *beta*",
+			Usage:       "Manage registry access *beta*",
```

Comment:
Ref. the `earthly doc` PR: it sounds like we historically [don't include `beta` tags in the CLI usage](https://github.com/earthly/earthly/pull/2579#discussion_r1083100766). Doesn't bother me to have this here, but it looks like it's inconsistent with other CLI commands that are in beta.

Reply:
ah ha, thanks for pointing that out.

Reply:
Sorry for the confusion previously, @nelsam. I think we do want to add `beta` tags in the end.

Reply:
Yeah, gotcha. I think that I saw a few instances in git history where the CHANGELOG mentioned sub-commands in beta, but the `usage` text didn't mention anything - IIRC that's what I was going on when I asked. But I don't think it's a bad idea to include those tags. I probably just found a few instances where they weren't applied consistently.